mreg/api/v1/zonefile.py

```diff
@@ -98,6 +98,18 @@ def mx_zf_string(self, name, ttl, priority, mx):
         }
         return '{name:24} {ttl:5} IN {record_type} {priority:6} {mx:39}\n'.format_map(data)
 
+    def sshfp_zf_string(self, name, ttl, algorithm, hash_type, fingerprint):
+
+        data = {
+            'name': name,
+            'ttl': ttl,
+            'record_type': "SSHFP",
+            'algorithm': algorithm,
+            'hash_type': hash_type,
+            'fingerprint': f'{fingerprint}'
```

Comment:
Uneeded f-string.

Reply:
You mean it can be just
'fingerpring': fingerprint ?

Reply:
Yes